src/js/app/components/profile/licenses/License.jsx

```diff
@@ -23,35 +23,39 @@
 
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { licenseRow, licenseStatus, nameLabel, sharingLabel } from 'css/license';
+import { licenseRow, nameLabel, sharingLabel } from 'css/license';
 
 class License extends Component {
-    static propTypes = {
-        codeSharing: PropTypes.string.isRequired,
-        name: PropTypes.string.isRequired,
-        licenseUrl: PropTypes.object.isRequired,
-        licenseTypeUrl: PropTypes.string.isRequired
-    };
 
     getSharingLabel() {
-        const { licenseTypeUrl } = this.props;
-        const sharingLabel = this.props.codeSharing
+        const { licenseTypeUrl, licenseFamilySummary } = this.props;
+        let finalLicenseTypeLabel = "";
+        let finalLicenseTypeURL = "";
+        if (this.props.codeSharing) {
+            finalLicenseTypeURL = licenseTypeUrl;
+            finalLicenseTypeLabel = this.props.codeSharing
             .split('_')
             .map(str => str[0] + str.slice(1)
-                .toLocaleLowerCase())
+            .toLocaleLowerCase())
             .join(' ');
+        }
+        if (licenseFamilySummary && licenseFamilySummary.href && licenseFamilySummary.name) {
```

Comment:
I believe there is more than one of these so we may need to account for that.

Reply:
there are multiple license object each with a single license family. So this isn't an issue